components/ee/payment-endpoint/src/chargebee/subscription-handler.ts

```diff
@@ -4,30 +4,30 @@
  * See License.enterprise.txt in the project root folder.
  */
 
```

Comment:
Sorry for all the formatter changes. seems like this component didn't get the big formatting update last time.